tests/test_fieldset.py

```diff
@@ -328,14 +333,17 @@ def test_fieldset_diffgrids_from_file(tmpdir, filename='test_subsets'):
     files = {'U': ufiles, 'V': vfiles}
     variables = {'U': 'vozocrtx', 'V': 'vomecrty'}
     dimensions = {'lon': 'nav_lon', 'lat': 'nav_lat'}
-    chs = 'auto'
 
-    fieldset = FieldSet.from_netcdf(files, variables, dimensions, timestamps=timestamps, allow_time_extrapolation=True, field_chunksize=chs)
-    assert fieldset.gridset.size == 2
-    assert fieldset.U.grid != fieldset.V.grid
+    fieldset = FieldSet.from_netcdf(files, variables, dimensions, timestamps=timestamps, allow_time_extrapolation=True, field_chunksize=field_chunksize)
+    if field_chunksize == 'auto':
+        assert fieldset.gridset.size == 2
+    else:
+        assert fieldset.gridset.size == 2
+        assert fieldset.U.grid != fieldset.V.grid
```

Comment:
in this case, both cases are exactly the same. We have two variables, so if we have two grids the grids automatically need to be different, isn't that so ?